lib/sequenceserver/blast/error.rb

```diff
@@ -0,0 +1,43 @@
+require 'English'
+
+require_relative 'constants'
+require_relative 'exceptions'
+
+module SequenceServer
+
+  #BLAST module
+  module BLAST
+    class Error
+      def initialize(job)
+        @job = job
+        call_error
+      end
+
+      attr_reader :job
+
+      def call_error
+        case job.exitstatus
+        when 1 # error in query sequence or options; see [1]
+          efile = File.open(job.efile)
```

Comment:
Simplify this code:
- Use File.readlines to read all lines at once.
- Use Array#grep to select the lines matching ERROR_LINE.
- Show matching lines if something matched or all lines in efile.